### ISSUE TITLE
html-indent rule stops working with presence of ES6 string interpolation in a :style

### ISSUE THREAD

**Issue by danwenzel** [vuejs/eslint-plugin-vue#953]
**Tell us about your environment**	
* **ESLint version: 5.11.1**	
* **eslint-plugin-vue version: 5.2.3**	
* **Node version: 10.14.1**	

**Please show your full configuration:**

```js
module.exports = {
  plugins: [
    'jest',
    'cypress'
  ],
  extends: [
    'eslint:recommended',
    'plugin:vue/recommended',
  ],
  parserOptions: {
    parser: 'babel-eslint',
    ecmaVersion: 2017,
    sourceType: 'module'
  }
}
```

The `html-indent` rule is normally working:

![image](https://user-images.githubusercontent.com/11724146/64476848-985c6f80-d159-11e9-9f9e-86fe7383cde8.png)

```
  4:1  warning  Expected indentation of 6 spaces but found 7 spaces   vue/html-indent
  5:1  warning  Expected indentation of 4 spaces but found 10 spaces  vue/html-indent

✖ 2 problems (0 errors, 2 warnings)
  0 errors and 2 warnings potentially fixable with the `--fix` option.

✨  Done in 0.92s.
```

But if I add a bound style attribute with ES6 string interpolation used in a value, the `html-indent` rule stops working:

![image](https://user-images.githubusercontent.com/11724146/64476875-fdb06080-d159-11e9-945a-ceea3c3c6466.png)

```
✨  Done in 0.77s.
```

This seems to occur regardless of where this attribute is located, and other eslint rules are still working

**Comment by ota-meshi** [comment]
I tried this issue and it seems to work correctly.
Can you tell me the difference with your repository?

https://github.com/ota-meshi/eslint-plugin-vue-issue953

![image](https://user-images.githubusercontent.com/16508807/64669932-522b4700-d49e-11e9-824a-e49e13a4d168.png)

**Comment by danwenzel** [comment]
Thanks for digging in, @ota-meshi !

In my case, I'm using yarn.  It looks to be an issue with some transitive dependency related to babel-eslint and/or some @babel scoped dependency.  It's been super hard to reproduce, but I finally was able to on your project.

You should be able to see it by going to this branch: https://github.com/danwenzel/eslint-plugin-vue-issue953/tree/broken-yarnlock

... and then running:

```
rm -rf node_modules
yarn install
yarn eslint Test.vue
```

To get the bug fixed in my project, I ended up having to blow away my `node_modules` _and_ my `yarn.lock` and I _think_ I also had to bump `babel-eslint` from 10.0.2 to 10.0.3 before all that.  This might have something to do with the issues discussed in the 10.0.3 release: https://github.com/babel/babel-eslint/releases/tag/v10.0.3.

**Comment by danwenzel** [comment]
Closing this one, as it seems to be pretty edgy and can probably be fixed using the steps I outlined above
